src/engine/test/engine-test-utils/src/engine_handler/handler.py

```diff
@@ -60,14 +60,17 @@ def _wait_to_live(self) -> bool:
 
         return True
 
-    def start(self, log_file_path: str = "") -> None:
+    def start(self, log_level: str = "", log_file_path: str = "") -> None:
         """Starts the engine process
 
         Raises:
             Exception: If the engine process fails to start or exits with a non-zero code
         """
 
-        command = f"{self.binary_path} --config {self.configuration_path} server start"
+        if log_level:
+            command = f"{self.binary_path} --config {self.configuration_path} server -l {log_level} start"
+        else:
+            command = f"{self.binary_path} --config {self.configuration_path} server start"
```

Comment:
Since the cli will be removed, do not add parameters that can be set in the configuration file, do not change the function signature.